community/cassandra-cpp-driver/APKBUILD

```diff
@@ -2,18 +2,31 @@
 # Maintainer: Gennady Feldman <gena01@gmail.com>
 pkgname=cassandra-cpp-driver
 _pkgreal=cpp-driver
-pkgver=2.7.0
-pkgrel=1
+pkgver=2.8.0
+pkgrel=0
 pkgdesc="Cassandra CPP Driver"
 url="http://datastax.github.io/cpp-driver/"
 arch="x86_64 x86 ppc64le"
 license="Apache-2.0"
-depends=
+depends="libuv libressl2.6-libcrypto libressl2.6-libssl"
```

Comment:
abuild will automatically pick up these and add dependency via the `so:` prefixed package names. Please leave `depends` empty.